code/modules/paperwork/paper.dm

```diff
@@ -40,10 +40,14 @@
 
 /obj/item/weapon/paper/update_icon()
 	if(burn_state == ON_FIRE)
+		// if(!istype(src, /obj/item/weapon/paper/paperplane)) //placeholder for paperplane_fire code
 		icon_state = "paper_onfire"
 		return
 	if(info)
-		icon_state = "paper_words"
+		if(!istype(src, /obj/item/weapon/paper/paperplane))  //don't update paperplanes
```

Comment:
So /obj/item/weapon/paper/paperplane tries to call /obj/item/weapon/paper/paperplane/update_icon(), but since it isn't defined, /obj/item/weapon/paper/update_icon() is called instead.

If you redefine that proc at the /obj/item/weapon/paper/paperplane level, that code is called instead. By doing this and doing nothing in it, you skip the update_icon() code that is meant for normal paper.